app/build.gradle.kts

```diff
@@ -113,7 +113,7 @@ dependencies {
     implementation(libs.androidx.compose.runtime)
     implementation(libs.androidx.lifecycle.runtimeCompose)
     implementation(libs.androidx.compose.runtime.tracing)
-    implementation(libs.androidx.compose.material3.windowSizeClass)
+    implementation(libs.androidx.compose.material3.adaptive)
```

Comment:
Note that nav suite will be material3-navigation-suite

Reply:
Fixed